spec/acceptance/notification_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+require 'capybara/rails'
+require 'support/pages/home'
+require 'support/with_user'
+
+RSpec.describe 'notifications/disable/:id', type: :feature do
+  with_auth_mock
+
+  let(:user) { User.all.first }
+
+  context 'logged in' do 
+    with_logged_in_user
+
+    it 'notifies user they are unsubscribed' do 
+      visit 'notifications/disable'
```

Comment:
With more time - I would have implemented the `Pages::` pattern you use in the other specs